### ISSUE TITLE
PostgreSQL compatibility

### ISSUE THREAD

**Issue by boboldehampsink** [barrelstrength/craft-sprout-forms#223]
### Description

Upgrading to latest Sprout Forms on PostgreSQL fails with the following error:

```
Exception: SQLSTATE[42703]: Undefined column: 7 ERROR:  column "%sproutForms:%" does not exist
```

### Steps to reproduce

1. Install previous Sprout Forms on a Postgres db
2. Update to latest

### Additional info

- Craft version: 3.1.5
- PHP version: 7.3.1
- Database driver & version: PostgreSQL 10.6
- Plugins & versions: Sprout Forms

**Comment by boboldehampsink** [comment]
Changing https://github.com/barrelstrength/craft-sprout-forms/blob/v3/src/migrations/m190124_000000_form_fields_settings.php#L27 to 

```php
            ->andWhere(['like', 'context', 'sproutForms:'])
```

Fixes the issue

**Comment by andrelopez** [comment]
Hi @boboldehampsink 

We have added your fix in Sprout Forms v3.0.0-beta.38

Thank you!
